selectize/selectize.d.ts

```diff
@@ -562,6 +563,12 @@ declare namespace Selectize {
          * Clears the render cache. Takes an optional template argument (e.g. "option", "item") to clear only that cache.
          */
         clearCache(template?: string): void;
+        
+        /**
+         * When the `settings.placeholder` value is changed, the new placeholder will be displayed.
+         */
+        updatePlaceholder: () => void;
```

Comment:
for consistency, I think this should be written as:

```ts
updatePlaceholder(): void;
```

also, are you sure the return type is void? I know the docs don't say, but how does it work at runtime?